funding/manager.go

```diff
@@ -3206,9 +3206,8 @@ func (f *Manager) handleInitFundingMsg(msg *InitFundingMsg) {
 
 	// For anchor channels cap the initial commit fee rate at our defined
 	// maximum.
-	if commitType == lnwallet.CommitmentTypeAnchorsZeroFeeHtlcTx &&
+	if commitType.HasAnchors() &&
```

Comment:
Should also do a scan for other areas where we may be trying to compare directly against the commit type. AFAICT, this is actually a bug right/ Which may have caused us to _not_ actually clamp the fee rate for certain classes of anchor channels?

Reply:
> Should also do a scan for other areas where we may be trying to compare directly against the commit type.

I already did and they're all covered now. The "find references" tool from `gopls` was very helpful in doing so.

> AFAICT, this is actually a bug right/ Which may have caused us to not actually clamp the fee rate for certain classes of anchor channels?

Where is the bug here? Or is this comment meant for something else?

Reply:
Scratch the comment about the bug, I thought there were instances where we compared the actual _bit field_ we store as a chan type against that value. That's a diff type in `channeldb` though, as this is the `lnwallet` analogue.